src/pages/User/components/PullRequests/hooks/useFetchPullRequests.js

```diff
@@ -8,10 +8,19 @@ const ACTIONS = {
 
 const API_URL = process.env.REACT_APP_API_URL;
 
-const fetchPullRequests = (username) =>
-  fetch(`${API_URL}/prs?username=${username}`, {
-    method: 'GET',
-  }).then((response) => response.json());
+// const fetchPullRequests = (username) =>
+//   fetch(`${API_URL}/prs?username=${username}`, {
+//     method: 'GET',
+//   }).then((response) => response.json());
+
+// Code by Georgey
```

Comment:
Can you remove? You can add your name to https://github.com/jenkoian/hacktoberfest-checker/blob/master/public/humans.txt if you want additional credit.